src/components/element-editor/element-editor.tsx

```diff
@@ -8,32 +8,39 @@ import {
 } from "./element-editor.styles";
 import { Prefill } from "./prefill";
 import { Validation } from "./validation/validation";
+import { useEffect } from "react";
+import { useFormContext } from "react-hook-form";
+import { IBaseTextBasedFieldValues } from "src/schemas";
```

Comment:
imports should be organised in alphabetical order, you can do so via the vscode command palette